internal/cmd/oohelperd/http_test.go

```diff
@@ -142,3 +142,54 @@ func Test_httpMapFailure(t *testing.T) {
 		})
 	}
 }
+
+func TestParseAltSvc(t *testing.T) {
```

Comment:
I would refactor this test to be a [table driven test](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests). When you previously contributed to OONI this fact was not clear to me, but now I am aware of it and I am quite on board with the concept. As a hint, if you rename this tested function to something else, you can get VSCode to auto-generate a table driven test for you by right clicking on the function name in `http.go` and choosing "auto-generate test for function" (or something alike) in the contextual menu. (Well, at least, I am a bit lazy so usually I use this strategy to create table-driven tests.)

Reply:
Thanks for providing that resource!